flpproto.sh

```diff
@@ -1,5 +1,5 @@
 package: flpproto
-version: alibuild-compilable
+version: master
```

Comment:
I suggest you use an actual tag for the version, and if you need to develop things you use a devel package. While we added the possibility to specify branch early in alibuild development, this was source of issues, because it triggers a rebuild every time.